src/fights.ts

```diff
@@ -1535,13 +1539,38 @@ const freeRunFightSources = [
     () =>
       (get("gingerbreadCityAvailable") || get("_gingerbreadCityToday")) &&
       get("_gingerbreadCityTurns") + (get("_gingerbreadClockAdvanced") ? 5 : 0) === 19 &&
-      availableAmount($item`sprinkles`) > 5,
+      (availableAmount($item`sprinkles`) > 5 || haveOutfit(`gingerbread best`)),
     () => {
       propertyManager.setChoices({
         1203: 4, // Gingerbread Civic Center 5 gingerbread cigarettes
         1215: 1, // Gingerbread Civic Center advance clock
+        1209: 2, //enter the gallery at Upscale Midnight
+        1214: 1, //get High-End ginger wine
       });
-      adventureMacro($location`Gingerbread Civic Center`, Macro.abort());
+      const boozes = $items`elemental caiproska, moreltini, Dreadsylvanian grimlet, Hodgman's blanket, Sacramento wine, iced plum wine, splendid martini, Eye and a Twist, jar of fermented pickle juice, dirt julep, ambitious turkey, friendly turkey`;
+      const boozeVals = Array.from(boozes.values()).map((drink) => {
+        const buff = effectModifier(drink, "Effect");
+        const turnsPerUse = numericModifier(drink, "Effect Duration");
+        const meatDrop = numericModifier(buff, "Meat Drop");
+        const famWeight = numericModifier(buff, "Familiar Weight");
+        const buffValue = ((meatDrop + famWeight*25/10) * turnsPerUse * (baseMeat+750));
+        const advValue = (getAverageAdventures(drink) * get("valueOfAdventure"))
+        return {
+          booze: drink,
+          value: (garboValue(drink) + buffValue + advValue) / drink.inebriety,
```

Comment:
usually when we're consuming something, we prefer "mallPrice" to determine how much it costs to buy. garboValue is for computing how much we would sell it for